### DISCUSSION TITLE
Update script from v1 to v2

### DISCUSSION THREAD

**Discussion by JeffGort** [stevenilsen123/mac-keyboard-behavior-in-windows#15] [General]
Amazing that you shared this. It improves my workflow tremendously on a Windows laptop, while being used to a Macbook for work!

AutoHotKey is moving to v2 and the original script gives errors. 

Some awesome people created a v1 to v2 converter and it made this script work in v2 :)

Code is here: https://github.com/mmikeww/AHK-v2-script-converter

**Comment by stevenilsen123** [comment]
Thank you for sharing this! I'll get this updated soon!

**Reply by stevenilsen123** [reply]
Thanks again for this! Just released the 2.0 update!
